### DISCUSSION TITLE
Add server without specifying a protocol

### DISCUSSION THREAD

**Discussion by mm0x1** [jellyfin/jellyfin-vue#1695] [Features]
I added the ability for users to be able to add servers without specifying any protocol, as mentioned in issue [ #1688](https://github.com/jellyfin/jellyfin-vue/issues/1688). 

Pull Request: [#1694](https://github.com/jellyfin/jellyfin-vue/pull/1694)

### Original
- The user was not allowed to press the "connect" button if the server URL does not start with "http://" or "https://"

### Changes
- the REGEX expression in the mustBeUrl function has been modified to accept IP addresses without ports or protocols
- A default value of "http://" and a port of 8096 are added if they are not present in the URL.

### Example
123.101.5.1, https://123.101.5.1, and 123.101.5.1:8096 are all converted to http://123.101.5.1:8096 before there is an attempt to establish a connection. URLs that contain a protocol and port are left intact.

Also, this is my first ever contribution to an open-source project.

**Comment by ferferga** [comment]
This is already implemented since the introduction of @jellyfin/sdk
